src/locales/ar/translation.json

```diff
@@ -1,3 +1,247 @@
 {
-  "example": "مرحبا"
+  "example": "Hello",
+  "workingTogether": {
+    "title": "نعمل معًا لإنهاء الجوع",
+    "text": "وفقًا لبرنامج الغذاء العالمي ، 135 مليونًا يعانون من الجوع الحاد بسبب الصراعات التي من صنع الإنسان، تغير المناخ، والركود الاقتصادي. يمكن أن تضاعف جائحة كوفيد-19 هذا العدد الآن ، مما يعرض 130 مليون شخص إضافي لخطر المعاناة من الجوع الحاد بحلول نهاية عام 2020. مع احتمال وجود أكثر من ربع مليار شخص على شفا المجاعة ، يجب اتخاذ إجراءات سريعة لتوفير الغذاء والإغاثة .الإنسانية للمناطق الأكثر عرضة للخطر"
+  },
+  "navbar": {
+    "headers": {
+      "header0": "الصفحة الرئيسية",
+      "header1": "القسائم",
+      "header2": "المدونة",
+      "header3": "تسجيل خروج"
+    },
+    "languages": {
+      "lang0": " التركية",
+      "lang1": "الإنجليزية",
+      "lang2": "العربية"
+    }
+  },
+  "joinUs": {
+    "title": "انضم إلينا",
+    "description": "انضم إلى مهمتنا للقضاء على الجوع. يمكنك الانضمام بصفتك مالك مطعم حيث يمكنك إضافة قسائمك الخاصة حتى يتمكن المتبرعون من شرائها",
+    "button": "إنشاء حساب"
+  },
+  "footer": {
+    "headers": {
+      "header0": "الصفحة الرئيسية",
+      "header1": "القسائم",
+      "header2": "المدونة"
+    },
+    "summary": "خليها علينا هي منصة يمكن للمتبرعين من خلالها التبرع بقسائم الطعام للأشخاص المحتاجين من المطاعم المختارة.",
+    "location": "اسطنبول/تركيا",
+    "rights": "كل الحقوق محفوظة"
+  },
+  "youcanhelp": {
+    "headers": {
+      "textSection1": "إن التبرع للأسباب التي تهتم بها لا يفيد المؤسسات الخيرية نفسها فحسب ، بل يمكن أن يكون مجزيًا للغاية بالنسبة لك أيضًا",
+      "texSection2": "يتبرع ملايين الأشخاص للجمعيات الخيرية بشكل منتظم لدعم القضايا التي يؤمنون بها ، فضلاً عن التأثير الإيجابي لها على حياتهم ..",
+      "button": "تبرع"
+    }
+  },
+  "team": {
+    "headers": {
+      "gizemdescription": "مهندس برمجيات مبتدئ",
+      "bertacdescription": "مهندس برمجيات مبتدئ",
+      "ihabdescription": "مهندس برمجيات مبتدئ",
+      "cihandesription": "مهندس برمجيات مبتدئ"
+    }
+  },
+  "ourGoalsComp": {
+    "card": {
+      "title0": "أهدافنا",
+      "title1": "القضاء على الفقر بجميع أشكاله في كل مكان",
+      "title2": "القضاء على الجوع وتحقيق الأمن الغذائي ",
+      "title3": "ضمان حياة صحية وتعزيز الرفاهية للجميع في جميع الأعمار"
+    }
+  },
+  "ourSponsorsComp": {
+    "card": {
+      "title0": "شركاؤنا"
+    }
+  },
+  "blog": {
+    "blog0": {
+      "title": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Saepe.",
```

Comment:
these are no longer lorem so maybe we can translate whats written there

Reply:
I actually changed back to lorem, since blog posts are shown as to how they were written by the user.